src/commons/components/log-in-form/log-in-form.tsx

```diff
@@ -3,16 +3,18 @@ import React, { Component } from "react";
 // styles
 import './log-in-form.css'
 
-export default class LogInForm extends Component {
+export default class LogInForm extends Component<{}, {}> {
     render() {
         return <article className="log-in-form">
             <section >
                 <h3>Authorize</h3>
                 <form>
+                    {/* @ts-ignore */}
```

Comment:
you do not use {/* @ts-ignore */} here
so that TypeScript does not swear, replace the for with htmlFor